test/integration2/schema/alter.spec.js

```diff
@@ -119,7 +122,25 @@ describe('Schema', () => {
               expect(item_two.column_notNullable).to.be.null;
               expect(item_two.column_defaultToAndNotNullable).to.be.null;
               expect(tableAfter).to.equal(
-                'CREATE TABLE "alter_table" (`column_integer` integer, `column_string` varchar(255), `column_datetime` datetime, `column_defaultTo` integer, `column_notNullable` varchar(255), `column_defaultToAndNotNullable` datetime)'
+                'CREATE TABLE "alter_table" (`column_integer` integer, `column_string` varchar(255), `column_datetime` datetime, `column_defaultTo` integer, `column_notNullable` varchar(255), `column_defaultToAndNotNullable` datetime, `column_nullable` boolean NULL)'
+              );
+            });
+
+            it('removes an existing null constraint if a not null constraint is added to a column', async () => {
+              await knex.schema.alterTable('alter_table', (table) => {
+                table.boolean('column_nullable').notNullable().alter();
+              });
+
+              const tableAfter = (await knex.raw(QUERY_TABLE))[0].sql;
+
+              expect(
```

Comment:
this does not need to be awaited?

Reply:
It definitely does! My bad, I think I misunderstood how the Chai Promise library works.